pkgs/development/node-packages/composition.nix

```diff
@@ -6,12 +6,12 @@
 
 let
   nodeEnv = import ./node-env.nix {
-    inherit (pkgs) stdenv python2 util-linux runCommand writeTextFile;
+    inherit (pkgs) stdenv python2 utillinux runCommand writeTextFile;
```

Comment:
Until we update node2nix you need to revert the utillinux changes. Sorry about that.

Reply:
no worries, node2nix automatically made that change when I ran generate.sh and I was curious whether I should've included it